Gemfile.lock

```diff
@@ -618,7 +607,7 @@ GEM
       activerecord (>= 5.1)
       state_machines-activemodel (>= 0.8.0)
     stringex (2.8.5)
-    stripe (5.35.0)
+    stripe (5.34.0)
```

Comment:
This should be at 5.35.0 in master now, might have gotten mixed up in the rebase?

Reply:
We've done a gazillion rebases on this PR now to somehow figure this failing spec. So, quite possibly.